Content.Shared/Mech/Components/MechComponent.cs

```diff
@@ -93,6 +93,9 @@ public sealed class MechComponent : Component
     [DataField("equipmentWhitelist")]
     public EntityWhitelist? EquipmentWhitelist;
 
+    [DataField("pilotWhitelist")]
```

Comment:
add doc

Reply:
A whitelist for restricting pilot access.